inferno/callbacks.py

```diff
@@ -323,14 +330,16 @@ def table(self, row):
             floatfmt=self.floatfmt,
         )
 
-    def on_epoch_end(self, net, *args, **kwargs):
+    # pylint: disable=unused-argument
+    def on_epoch_end(self, net, **kwargs):
         data = net.history[-1]
         tabulated = self.table(data)
 
         if self.first_iteration_:
             header, lines = tabulated.split('\n', 2)[:2]
             self.sink(header)
             self.sink(lines)
+            # pylint: disable=attribute-defined-outside-init
```

Comment:
We need `first_iteration_` to be defined in `initialize` because if we re-initialize the net, it should be set to `True`, even if `__init__` is never called.

My guess is that since pylint does not know that `initialize` is always called before `on_epoch_end`, it believes that `first_iteration_` is defined here, when really it is only changed here.